### PR TITLE
Feature: Accent Rows

### PR DESCRIPTION
**Pull request by jacob-paulson**
Feature:

- Allows ids of rows to be passed into the ModelTableStore from the ModelTable component and accents them for focus if that id row is rendered

### CODE REVIEW

File: src/ModelTable/ModelTableStoreContext.tsx [hunk 1]
```diff
@@ -54,6 +54,7 @@ export interface ModelTableState<
   formOptions?: FormOptions;
   onUpdate?: OnActionTrigger<DT>;
   onDelete?: OnActionTrigger<DT>;
+  idToFocus?: string;
```
Comment (robxbob) [inline src/ModelTable/ModelTableStoreContext.tsx]: Rename to `selectedRows: ID[]`; `ID` is a type in conveyor

In the future, we will be able to select multiple rows.

File: src/ModelTable/ModelTableRow.tsx [hunk 2]
```diff
@@ -38,7 +39,11 @@ export const ModelTableRow = ({
   return (
     <FormStoreProvider id={data.id} {...formMethods}>
       <Lenses initialLens={DataLens.DISPLAY}>
-        <TableRow prefilled={false} {...props}>
+        <TableRow
+          prefilled={false}
+          className={`${data.id === idToFocus && 'bg-blue-200'}`}
```
Comment (robxbob) [inline src/ModelTable/ModelTableRow.tsx]: Choose a color variable or create a new color var.

File: src/ModelTable/ModelTableStoreContext.tsx (hunk 1 above)
Comment (jacob-paulson) [reply src/ModelTable/ModelTableStoreContext.tsx]: Sounds good, implemented!

File: src/ModelTable/ModelTableRow.tsx (hunk 2 above)
Comment (jacob-paulson) [reply src/ModelTable/ModelTableRow.tsx]: Ah there's even an `accent` color already set, changed! Thanks for the catch.

**Pull request merged**